product_docs/docs/pgd/5/cli/command_ref/pgd_verify-settings.mdx

```diff
@@ -73,18 +175,18 @@ pgd verify-settings [flags]
 
 
   # shared_preload_libraries
-  Node               Status  Pending Restart Value                   Message
-  ----               ------  --------------- -----                   -------
-  bdr-a1             Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  bdr-a2             Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  bdr-b1             Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  bdr-b2             Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  logical-standby-a1 Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  logical-standby-b1 Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  subscriber-only-c1 Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  witness-a          Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  witness-b          Warning false           pg_stat_statements, bdr must contain bdr as first entry
-  witness-c          Warning false           pg_stat_statements, bdr must contain bdr as first entry
+  Node               Status  Pending Restart Value         Message
+  ----               ------  --------------- -----         -------
+  bdr-a1             Warning false           pg_stat_stat… must contain bdr as first entry
+  bdr-a2             Warning false           pg_stat_stat… must contain bdr as first entry
+  bdr-b1             Warning false           pg_stat_stat… must contain bdr as first entry
+  bdr-b2             Warning false           pg_stat_stat… must contain bdr as first entry
+  logical-standby-a1 Warning false           pg_stat_stat… must contain bdr as first entry
+  logical-standby-b1 Warning false           pg_stat_stat… must contain bdr as first entry
+  subscriber-only-c1 Warning false           pg_stat_stat… must contain bdr as first entry
+  witness-a          Warning false           pg_stat_stat… must contain bdr as first entry
+  witness-b          Warning false           pg_stat_stat… must contain bdr as first entry
+  witness-c          Warning false           pg_stat_stat… must contain bdr as first entry
 
```

Comment:
It's the autogenerated output from pgd verify-settings.

Reply:
If you are talking about the truncated value of `pg_stat_statement`, then the value text is truncated to 15 chars to avoid the breakage of tabular output in case this string is too long. The complete value is visible in JSON output if the command is executed with the `-o json` flag.